modules/codegen/src/smithy4s/codegen/CodegenArgs.scala

```diff
@@ -28,3 +28,8 @@ case class CodegenArgs(
     dependencies: List[String],
     transformers: List[String]
 )
+
+case class ProtoTypeArgs(
+                          smithyString:String,
```

Comment:
Could we achieve the same outcome by using the `codegen` command with `-` as the input path, and piping the string through stdin?